.rubocop.yml

```diff
@@ -57,33 +57,61 @@ SingleLineMethods:
   AllowIfMethodIsEmpty: false
 SpaceInsideHashLiteralBraces:
   EnforcedStyle: no_space
+TrivialAccessors:
+  AllowPredicates: true
 #
 # Enabled/Disabled
 #
 AllCops:
   RunRailsCops: true
+Alias:
+  Enabled: false
 ClassAndModuleChildren:
   Enabled: false
+DefEndAlignment:
+  AutoCorrect: true
 Documentation:
   Enabled: false
 Encoding:
   Enabled: false
+EndAlignment:
+  AutoCorrect: true
+ExtraSpacing:
+  AutoCorrect: false # https://github.com/bbatsov/rubocop/issues/2280
+FindEach:
+  Enabled: false
 GuardClause:
   Enabled: false
 IfUnlessModifier:
   Enabled: false
+NumericLiterals:
+  Enabled: false
```

Comment:
I'm on the fence on this one because in some cases I love it, and it other cases I hate it.